src/pages/Login/LoginForm/LoginInput.tsx

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
create types.ts in  src/pages/Login/LoginForm, and put types that you need use in elsewhere. Don't remember export it from src/pages/Login/LoginForm/index.ts

Reply:
thanks